docs/modules/ROOT/pages/runbooks/CephOSDSlowOps.adoc

```diff
@@ -0,0 +1,29 @@
+= Alert rule: CephOSDSlowOps
+
+== icon:glasses[] Overview
+
+This alert fires if some Ceph OSD requests take too long to process.
```

Comment:
Whats the concequence of this? Do IO on volumes drop or something else?

Reply:
Expanded the description for the alert